src/main/java/io/datanerds/avropatch/operation/DefaultSchema.java

```diff
@@ -0,0 +1,84 @@
+package io.datanerds.avropatch.operation;
+
+/**
+ * This interface holds the schema for all valid value types.
+ */
+public interface DefaultSchema {
+    String VALUE =
```

Comment:
This would not compile per se since the value is used in an annotation (only constants allowed there). Would need an indirection for compilation. I am adding a card to the backlog